content/html/concepts/semantic-html/semantic-html.md

```diff
@@ -33,8 +39,8 @@ With Semantic HTML, we were introduced to elements that tell developers and brow
 
 ## Why Use Semantic HTML?
 
-- Accessibility: Semantic HTML makes webpages accessible for mobile devices and for people with disabilities as well. This is because screen readers and browsers are able to interpret the code better.
+- Accessibility: Semantic HTML makes webpages accessible for mobile devices and for people with disabilities as well. This is because screen readers and browsers are better able to interpret the content of the site.
 
 - Readability: Semantic HTML also makes the website’s source code easier to read for other web developers.
 
-- SEO: It improves the website SEO, or Search Engine Optimization, which is the process of increasing the number of people that visit your webpage. With better SEO, search engines are better able to identify the content of your website and weight the most important content appropriately.
+- SEO: It improves the website SEO, or Search Engine Optimization, which is the process of increasing the number of people that visit your webpage. With better SEO, search engines are better able to identify the content of your website and weigh the most important content appropriately.
```

Comment:
```suggestion
- SEO: It improves the website SEO, or Search Engine Optimization, which is the process of increasing the number of people that visit a webpage. With better SEO, search engines are better able to identify the content of a website and weigh the most important content appropriately.
```
Getting rid of the second-person pronoun.

Reply:
@SSwiniarski All set. Thanks